ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -340,6 +284,10 @@ objects:
           echo "Removing ssh-bastion"
           oc delete project openshift-ssh-bastion
           oc wait namespace/openshift-ssh-bastion --for delete --timeout=10m
+
+          echo "Remove existing openshift-apiserver pods"
+          # This would ensure "Pod 'openshift-apiserver/apiserver-xxx' is not healthy: container openshift-apiserver has restarted more than 5 times" test won't fail
+          oc delete pod --all -n openshift-apiserver
```

Comment:
hmm, this probably also blows away our logs for those pods?  Maybe we want to pull down their logs into the shared artifacts volume before doing this?